src/main/scala/io/iohk/ethereum/governance/CertificateAuthorityManager.sol

```diff
@@ -6,8 +6,8 @@ contract CertificateAuthorityManager {
 
 	address a1 = 0x00edde8656c35fcb7126c61fc6e2673734425a72bf;
 
-	function isCertificateAuthorityFor(address sender, uint slotNumber) constant returns(bool) {
-		CAQueriedFor(sender, slotNumber);
+	function isCertificateAuthorityFor(address sender) constant returns(bool) {
+		CAQueriedFor(sender, 0);
```

Comment:
Minor detail: Maybe we can remove the parameter from the event that will always be 0?

Reply:
this will be fix in the next PR! =)